inspire_disambiguation/api.py

```diff
@@ -76,34 +78,46 @@ def train_and_save_distance_model(
         ethnicity_model_path (str): Full path where ethnicity model is saved.
         save_distance_model_path (str): Full path where trained distance model
             will be saved.
-        sampled_pairs_size (int): Number of pairs to be generated for the training.
+        sampled_pairs_size (int): Number ożf pairs to be generated for the training.
             Note:
                 Must be multiple of 12.
     """
     LOGGER.info("Pulling training data from ES")
     start_time = datetime.now()
     curated_signatures = get_signatures(only_curated=True)
-    input_clusters = get_input_clusters(curated_signatures)
+    curated_signatures_dict = load_signatures(curated_signatures)
+    train_signatures_uuids = random.sample(list(curated_signatures_dict),
+                                           k=int(len(curated_signatures) * 0.8))
+    train_signatures = [curated_signatures_dict[uuid] for uuid in train_signatures_uuids]
+    test_signatures_uuids = set(list(curated_signatures_dict)) - set(list(curated_signatures_dict))
+    test_signatures = [curated_signatures_dict[uuid] for uuid in test_signatures_uuids]
+    input_clusters_train = get_input_clusters(train_signatures)
+    input_clusters_test = get_input_clusters(test_signatures)
```

Comment:
maybe split this whole thing off into a separate helper function? I would also make the splitting fraction configurable (and use more training data probably, like 90%, as this is purely for evaluation, we're not tuning any hyper-parameters here).